inc/namespace.php

```diff
@@ -43,6 +45,37 @@ function admin_bar_render() {
 	die();
 }
 
+/**
+ * Set a cookie to identify that the current user is logged in.
+ *
+ * We can't rely on WordPress's LOGGED_IN_COOKIE, which is HTTP-only.
+ */
+function set_js_cookie() {
+	setcookie(
+		'wp_remote_admin_bar',
+		1,
+		[
+			'domain'   => COOKIEDOMAIN,
+			'path'     => SITE_COOKIE_PATH,
+			'httponly' => false
+		] );
+}
+
+/**
+ * Clear the cookie used to identify logged in users.
+ */
+function clear_js_cookie() {
+	setcookie(
+		'wp_remote_admin_bar',
+		0,
+		[
+			'domain'  => COOKIEDOMAIN,
+			'path'    => SITE_COOKIE_PATH,
+			'expires' => time() - YEAR_IN_SECONDS,
+		]
+	);
```

Comment:
This is PHP 7.3 syntax so that will need adding as the minimum version requirement in the readme and in the plugin header.